src/views/Viewer.vue

```diff
@@ -907,13 +913,15 @@ export default {
 
 		/**
 		 * Show the sharing sidebar
+		 *
+		 * @param {boolean} openSidebar open the sidebar, defaults to true
 		 */
 
-		async showSidebar() {
+		async showSidebar(openSidebar = false) {
 			// Open the sidebar sharing tab
 			// TODO: also hide figure, needs a proper method for it in server Sidebar
 
-			if (OCA?.Files?.Sidebar) {
+			if (OCA?.Files?.Sidebar && openSidebar) {
```

Comment:
This means we will never be able to open the sidebar on small screens

Reply:
Yeah, it was just a test but didnt work anyway. I will have a look together with @Pytal on the problem. Or if you have some suggestions, please note them down :)